src/engraving/dom/segment.h

```diff
@@ -304,7 +304,8 @@ class Segment final : public EngravingItem
 
     void checkEmpty() const;
 
-    static constexpr SegmentType durationSegmentsMask = SegmentType::ChordRest | SegmentType::TimeTick;   // segment types which may have non-zero tick length
+    static constexpr SegmentType CHORD_REST_OR_TIME_TICK_TYPE = SegmentType::ChordRest | SegmentType::TimeTick;
```

Comment:
Just a small one - would we like to keep this near the `SegmentType` definition?  It could take the place of `constexpr SegmentType SPANNER_ANCHOR_SEG_TYPE` which has outlived any of its uses and needs to be removed anyway.

Reply:
Ah yes, good point I'll remove it. Though I do prefer actually to keep this as a static member of segment, cause I think accessing it as `Segment::CHORD_REST_OR_TIME_TICK_TYPE` makes it clearer what it is from the outside (as opposed to just `CHORD_REST_OR_TIME_TICK_TYPE`)